src/controllers/user.controller.js

```diff
@@ -0,0 +1,100 @@
+import { ApiError } from '../exception/api.error.js';
+import { userService } from '../services/user.service.js';
+
+const getAllActivated = async (req, res) => {
+  const users = await userService.getAllActivated();
+
+  res.send(users.map(userService.normalize));
+};
+
+function validateName(value) {
+  if (!value) {
+    return 'Name is required';
+  }
+
+  if (value.includes(' ')) {
+    return 'One word required';
+  }
+
+  if (value.length < 2) {
+    return 'At least 2 characters';
+  }
+}
+
+const validatePassword = (value) => {
+  if (!value) {
+    return 'Password is required';
+  }
+
+  if (value.length < 6) {
+    return 'At least 6 characters';
+  }
+};
+
+function validateEmail(value) {
+  if (!value) {
+    return 'Email is required';
+  }
+
+  const emailPattern = /^[\w.+-]+@([\w-]+\.){1,3}[\w-]{2,}$/;
+
+  if (!emailPattern.test(value)) {
+    return 'Email is not valid';
+  }
+}
+
+const changeName = async (req, res) => {
+  const { newName } = req.body;
+  const { userId } = req.params;
+
+  if (validateName(newName)) {
+    throw ApiError.badRequest(validateName(newName));
```

Comment:
The function `validateName` returns a string message when validation fails, which is a truthy value. Therefore, the condition `if (validateName(newName))` will always be true if the name is invalid. You should check if the return value is not undefined or null instead.